src/debug/GdbCmds.h

```diff
@@ -1,4 +1,4 @@
-
+#include <cstddef>
```

Comment:
GdbCmds.h is only every supposed to be included in gdb stub, so the include here is pointless.

Reply:
got it. ill remove that part of the patch (on the other hand, what a weird way to include code)